src/auth-service/config/global/email-templates.js

```diff
@@ -78,8 +78,25 @@ const emailTemplates = {
                                             style="color: #667085; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">This
                                             email was sent to</span>
                                         <span
-                                            style="color: #135DFF; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">${email}</span>
+                                            style="color: #135DFF; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">${email}<br/></span>
+                                            
+                                        ${this.EMAIL_UNSUBSCRIBE_CLAUSE(optional)}
```

Comment:
thanks @BenjaminSsempala , have we carried out extensive tests with "this" keyword? Sometimes, it can do "bad manners" if not properly reviewed. :)

Reply:
@Baalmart Yes, I've tested it out.

Reply:
Awesome! Thanks @BenjaminSsempala